rpcs3/Crypto/unzip.cpp

```diff
@@ -3,6 +3,8 @@
 
 #include <zlib.h>
 
+#include "Emu/savestate_utils.hpp"
```

Comment:
If you need to include this here, something is really wrong with your software design

Reply:
More like the naming of the file.